frontend/src/v5/ui/routes/viewer/drawings/drawingsList/drawingItem/drawingItem.component.tsx

```diff
@@ -30,10 +32,17 @@ type DrawingItemProps = {
 	onClick: React.MouseEventHandler<HTMLDivElement>;
 };
 export const DrawingItem = ({ drawing, onClick }: DrawingItemProps) => {
-	const [latestRevision] = DrawingRevisionsHooksSelectors.selectRevisions(drawing._id);
-	const { calibration, name, drawingNumber, lastUpdated, desc } = drawing;
+	const params = useParams();
+	const history = useHistory();
+	const { calibration, name, drawingNumber, lastUpdated, desc, _id: drawingId } = drawing;
+	const [latestRevision] = DrawingRevisionsHooksSelectors.selectRevisions(drawingId);
 	const { statusCode, revisionCode } = latestRevision || {};
 	const areStatsPending = !revisionCode;
+	
+	const onCalibrateClick = () => {
```

Comment:
Would it make sense to move this into the DrawingsCalibrationMenu component since component already requires the drawingId

Reply:
Clicking on the calibration button may have 2 different outcomes: 
- from the drawings list in the viewer (`src\v5\ui\routes\viewer\drawings\drawingsList\drawingItem\drawingItem.component.tsx`), it should just append the drawingId in the URL as a search param
- from the list in the dashboard (`\src\v5\ui\routes\dashboard\projects\drawings\drawingsList\drawingsListItem\drawingsListItem.component.tsx`), it should first pop-up a modal to let the user select the con/fed, then, redirect to the viewer